### PR TITLE
[10.0] mig account_due_list

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

**Comment by vonpupp** [comment]
I installed the module and it seems to install fine.

File: account_due_list/README.rst [hunk 1]
```diff
@@ -1,16 +1,15 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
-Account due list
-================
+=====================
+Payments and due list
```
Comment (elicoidal) [inline account_due_list/README.rst:6]: Payment due list?

File: account_due_list/__manifest__.py [hunk 2]
```diff
@@ -1,42 +1,32 @@
...
+# © 2015 Tecnativa (http://www.tecnativa.com)
+# © 2016 Eficent Business and IT Consulting Services S.L.
+#        (http://www.eficent.com)
+# © 2016 David Dufresne <david.dufresne@savoirfairelinux.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
 {
     'name': "Payments Due list",
-    'version': '8.0.0.1.0',
+    'version': '10.0.0.1.0',
```
Comment (elicoidal) [inline account_due_list/__manifest__.py]: 10.0.1.0.0

File: account_due_list/views/payment_view.xml [hunk 3]
```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
```
Comment (elicoidal) [inline account_due_list/views/payment_view.xml]: replace openerp by odoo

File: account_due_list/views/payment_view.xml [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+                       sum="Total Residual Currency"/>
+                <field name="date_maturity"/>
+                <field name="move_id" readonly="1"/>
+                <field name="reconciled" readonly="1"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_payments_filter" model="ir.ui.view">
+        <field name="name">Payments Select</field>
```
Comment (elicoidal) [inline account_due_list/views/payment_view.xml]: Selection?

**Review by elicoidal — CHANGES_REQUESTED** [review]
some details but overall LGTM

File: account_due_list/README.rst (hunk 1 above)
Comment (eLBati) [reply account_due_list/README.rst:6]: This is actually not changed between 9.0 and 10.0

You can  see them with
```
git diff origin/9.0..eLBati/10.0-mig-account_due_list  account_due_list
```

File: account_due_list/__manifest__.py (hunk 2 above)
Comment (eLBati) [reply account_due_list/__manifest__.py]: I would rather change it at merging time, in order to better handle versions with setuptools-odoo

File: account_due_list/views/payment_view.xml (hunk 3 above)
Comment (eLBati) [reply account_due_list/views/payment_view.xml]: Thanks, done

File: account_due_list/views/payment_view.xml (hunk 4 above)
Comment (eLBati) [reply account_due_list/views/payment_view.xml]: Thanks, done

File: account_due_list/__manifest__.py (hunk 2 above)
Comment (eLBati) [reply account_due_list/__manifest__.py]: Nevermind , I changed it anyway

**Comment by vonpupp** [comment]
I have created a simple [buildout] for testing purposes. Please use the branch `10-mig-account_due_list`. I tested the installation and it seems ok.

[buildout]: https://github.com/vonpupp/testpr-buildout

**Review by vonpupp — APPROVED** [review]

**Comment by vonpupp** [comment]
Any update on this PR?

**Comment by pedrobaeza** [comment]
OK, I think we can merge.

**Comment by vonpupp** [comment]
Great. Thank you very much @eLBati and @pedrobaeza.

**Pull request merged**